From 4057c75e70bea274fd0fcabf0e02e4607134dd79 Mon Sep 17 00:00:00 2001
From: TheAtt1la <alittauwu@gmail.com>
Date: Sun, 18 Dec 2022 15:13:52 -0300
Subject: [PATCH] remove duplicate ro.adb.secure

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 031e160..fc8d9b6 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,7 +18,7 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.38.1

